src/Hazelcast.Net.Tests/DotNet/ExceptionTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Runtime.ExceptionServices;
+using NUnit.Framework;
+using Hazelcast.Core;
+
+namespace Hazelcast.Tests.DotNet
+{
+    [TestFixture]
+    public class ExceptionTests
+    {
+        [Test]
+        public void MustThrowToGetStackTrace()
```

Comment:
What is the goal here? Do we expect something different between .net versions? What am I missing?